design-guidelines/form-ux.md

```diff
@@ -0,0 +1,189 @@
+# Form Design Guidelines
+
+A form may be used to collect data that is required to complete a task.
+
+## Form contexts
+
+The context in which a form is rendered depends on the complexity of the form and how critical the form's purpose is to the rest of the app.
+
+### Full page
+
+A form may be rendered on it's own page helps users to focus on completing their task with minimal distractions in the UI.
+
+Render a form on it's own page if the form:
```

Comment:
is there any way to reduce subjectivity? like providing examples from dc/os and ksphere

Reply:
Sure, I could add examples like I did in the page layout patterns.